org.nifty-site-manager.nsm.appdata.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2019 Nick Ham -->
+<component type="console-application">
+  <id>org.nifty-site-manager.nsm</id>
```

Comment:
This app-id is invalid. It will have to be renamed to `org.nifty_site_manager.nsm`

Reply:
Actually you don't even control this domain the correct app-id is: `cc.nift.nsm`